### PR TITLE
Create About Us Alumni Highlights Section

### LINKED ISSUE #20: Create About Us Page Highlights Section
image

### PR DESCRIPTION
**Pull request by tylerlam1**
Resolves #20.

Notes:
- Sophia asked to change her picture on the website, so I just did it in this PR

Feedback for Topan:
- Great work! I just changed some tailwind dimensions from px to rem to keep it consistent - and let's use rem if possible because I think it's technically better
- We don't need to use the fragment `<>` if there is only one child element in a component
- No glaring issues I saw

**Webpage**
image

**Mobile**
image

### CODE REVIEW

File: components/AlumniCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const AlumniTile = ({ name, file, position, prevRole, testimonial }) => {
+  return (
+    <div className={ALUMNI_TILE}>
+      <div className={ALUMNI_PROFILE}>
+        <img src={`/profiles/${file}`} alt="Profile Picture" className="w-50 mb-3 mt-5 rounded-full md:w-64 mx-auto" />
+        <h2 className="text-[2rem] font-bold">{name}</h2>
+        <h2 className="text-[1.25rem] font-semibold text-[#BAFBE4]">{position}</h2>
+        <h2 className="text-[1.25rem] font-medium mb-5">Previously: {prevRole}</h2>
+      </div>
+      <p className="text-[1.3rem] font-light italic leading-7 mx-auto md:mb-24 md:text-[1.5rem]">
```
Comment (harsohailB) [inline components/AlumniCarousel.jsx]: Can we make this bigger on desktop. Feels like there is a lot of white space. At least on high resolutions

File: components/AlumniCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import "swiper/css/navigation";
+
+const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[37.5rem] shadow-2xl md:w-4/5 swiper-button-white";
+const ALUMNI_TILE =
+  "bg-[#7055FD] w-[20rem] h-[37.5rem] rounded-lg items-center p-5 flex flex-col shadow-2xl md:w-full md:flex-row";
+const ALUMNI_PROFILE = "flex flex-col items-center mx-auto";
+
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const AlumniTile = ({ name, file, position, prevRole, testimonial }) => {
```
Comment (harsohailB) [inline components/AlumniCarousel.jsx]: Can we maintain padding on resize between the elements?

Screen Shot 2022-08-21 at 10 56 55 PM

Comment (harsohailB) [inline components/AlumniCarousel.jsx]: Add more padding between arrows and content on mobile?

Screen Shot 2022-08-21 at 10 57 28 PM

**Review by harsohailB — CHANGES_REQUESTED** [review]
Close! Just minor CSS nits about padding

**Comment by harsohailB** [comment]
Random: Should we add linkedin for alumnis if people want to reach out?

File: components/AlumniCarousel.jsx (hunk 1 above)
Comment (tylerlam1) [reply components/AlumniCarousel.jsx]: Gave it a shot. Take a look

File: components/AlumniCarousel.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const ALUMNI_PROFILE = "flex flex-col items-center mx-auto";
+
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) => {
+  return (
+    <div className={ALUMNI_TILE}>
+      <div className={ALUMNI_PROFILE}>
+        <img src={`/profiles/${file}`} alt="Profile Picture" className="w-50 mb-3 mt-5 rounded-full md:w-80 mx-auto" />
+        <h2 className="text-[2rem] md:text-[2.5rem] font-bold">{name}</h2>
```
Comment (harsohailB) [inline components/AlumniCarousel.jsx:23]: These dont seem to centered

Screen Shot 2022-08-23 at 10 14 59 PM

**Review by harsohailB — CHANGES_REQUESTED** [review]

File: components/AlumniCarousel.jsx (hunk 3 above)
Comment (tylerlam1) [reply components/AlumniCarousel.jsx:23]: Check now

**Review by harsohailB — APPROVED** [review]

**Pull request merged**